__init__.py

```diff
@@ -504,22 +504,31 @@ def play_song(self, message):
         artist = message.data.get('artist')
         album = message.data.get('album')
 
-        # workaround for Padatious training, as the most generic "play {track}" is taking precedence over the play_something
-        # and play_playlist rules
+        # workaround for Padatious training, as the most generic "play {track}"
+        # is taking precedence over the play_something and play_playlist rules
         if song and not album:
-            m = re.match(r'^play( the| my)? playlist (?P<playlist>[\w\s]+?)$' , message.data['utterance'], re.M|re.I)
+            if song == 'spotify':
+                self.continue_current_playlist(message)
+            LOG.info(self.translate('playlist_regex'))
+            m = re.match(self.translate('playlist_regex'),
+                         message.data['utterance'], re.M | re.I)
             if m:
-                LOG.info("I'm in the play_song handler but I've seen an utterance that contains 'playlist.' I want to play the playlist " + m.group(1) + ". Switching handlers.")
+                LOG.debug('I\'m in the play_song handler but I\'ve seen'
+                          ' an utterance that contains \'playlist.\''
```

Comment:
Minor thing, but is it really accepted style to use single quotes for English strings that include apostrophes? It makes it so cumbersome, as you have to escape them all the time

Reply:
This is a personal quirk, I prefer using `'` even if it's messier. I rationalize it with being more consistent or helping me internally differentiate between python and C... But in reality it's just a habit.